carina-core/src/main/java/com/qaprosoft/carina/core/foundation/listeners/AbstractTestListener.java

```diff
@@ -622,7 +623,7 @@ protected void setConfigFailure(TestResultItem resultItem) {
         configFailures.set(resultItem);
     }
 
-    private String takeScreenshot(ITestResult result, String msg) {

```

Comment:
@ivansnytko how about making it private again and moving to CarinaListener class?